group2_assignment2_solution.ipynb

```diff
@@ -426,20 +443,23 @@
     "'''\n",
     "# INSERT YOUR CODE HERE\n",
     "\n",
+    "print(k1_tweets[0].keys())\n",
```

Comment:
I'm happy with this, probably no need to loop through them on second thought. The purpose is to examine the keys and we can do that easily with this method 👍